.changeset/proud-zebras-jump.md

```diff
@@ -0,0 +1,7 @@
+---
+"react-native-reanimated-carousel": major
+---
+
+feat: use new RNGH api
+
+Updates `react-native-gesture-handler` to `>2.9.0` and replaces usage of `useAnimatedGestureHandler` with the [new gesture handler API](https://docs.swmansion.com/react-native-gesture-handler/docs/#rngh-20) which supports the [new gesture handler web implementation](https://github.com/software-mansion/react-native-gesture-handler/pull/2157).
```

Comment:
`>2.9.0` this line should have greater than equal to rather than only greater then like `>=2.9.0`